src/components/transaction/TransactionTable/index.tsx

```diff
@@ -0,0 +1,294 @@
+                {getShortOrderId(shortId)}
+              </LinkWithPrefixNetwork>
+            }
+          />
+        </HeaderValue>
+      </td>
+      <td>
+        <HeaderTitle>Type</HeaderTitle>
+        <span className="header-value">
+          <TradeOrderType kind={kind || 'sell'} />
+        </span>
+      </td>
+      <td>
+        <HeaderTitle>Sell Amount</HeaderTitle>
+        <HeaderValue>
+          {renderSpinnerWhenNoValue(sellFormattedAmount) || (
+            <TextWithTooltip textInTooltip={`${sellFormattedAmount} ${sellTokenSymbol}`}>
+              {formattedAmount(sellToken, sellAmount, FormatAmountPrecision.highPrecision)}{' '}
+              {sellToken && network && <TokenDisplay showAbbreviated erc20={sellToken} network={network} />}
+            </TextWithTooltip>
+          )}
+        </HeaderValue>
+      </td>
+      <td>
+        <HeaderTitle>Buy amount</HeaderTitle>
+        <HeaderValue>
+          {renderSpinnerWhenNoValue(buyFormattedAmount) || (
+            <TextWithTooltip textInTooltip={`${buyFormattedAmount} ${buyTokenSymbol}`}>
+              {formattedAmount(buyToken, buyAmount, FormatAmountPrecision.highPrecision)}{' '}
+              {buyToken && network && <TokenDisplay showAbbreviated erc20={buyToken} network={network} />}
+            </TextWithTooltip>
+          )}
+        </HeaderValue>
+      </td>
+      <td>
+        <HeaderTitle>
+          Limit price
+          <Icon icon={faExchangeAlt} onClick={invertLimitPrice} />
+        </HeaderTitle>
+        <HeaderValue>{renderSpinnerWhenNoValue(limitPriceSettled) || limitPriceSettled}</HeaderValue>
+      </td>
+      <td>
+        <HeaderTitle>Created</HeaderTitle>
+        <HeaderValue>
+          <DateDisplay date={expirationDate} showIcon={true} />
+        </HeaderValue>
+      </td>
+      <td>
+        <HeaderTitle>Status</HeaderTitle>
+        <HeaderValue>
+          <StatusLabel status={order.status} partiallyFilled={partiallyFilled} />
+        </HeaderValue>
+      </td>
+    </tr>
+  )
+}
+
+const TransactionTable: React.FC<Props> = (props) => {
+  const { orders, showBorderTable = false } = props
+  const [isPriceInverted, setIsPriceInverted] = useState(false)
+  useEffect(() => {
+    setIsPriceInverted(isPriceInverted)
+  }, [isPriceInverted])
+  const invertLimitPrice = (): void => {
+    setIsPriceInverted((previousValue) => !previousValue)
+  }
+
+  const orderItems = (items: Order[] | undefined): JSX.Element => {
+    let tableContent
+    if (!items || items.length === 0) {
+      tableContent = (
+        <tr className="row-empty">
+          <td className="row-td-empty">
+            <EmptyItemWrapper>
+              Can&apos;t load details <br /> Please try again
+            </EmptyItemWrapper>
+          </td>
+        </tr>
+      )
+    } else {
+      tableContent = (
+        <>
+          {items.map((item, i) => (
+            <RowTransaction
+              key={`${item.shortId}-${i}`}
```

Comment:
Curious why the `${i}` was needed here. Can't we just identify the order by its ID alone? Wouldn't this be better?